loris/img.py

```diff
@@ -215,21 +215,21 @@ def __getitem__(self, image_request):
 		return fp
 
 	@staticmethod
-	def _link(to,frum):
+	def _link(to, frum):
 		link_dp = path.dirname(frum)
 		if not path.exists(link_dp):
 			makedirs(link_dp)
 		if path.lexists(frum): # shouldn't be the case, but helps debugging
 			unlink(frum)
-		symlink(to,frum)
-		logger.debug('Made symlink from %s to %s' % (to,frum))
+		symlink(to, frum)
+		logger.debug('Made symlink from %s to %s' % (frum, to))
 
-	def __setitem__(self, image_request, fp):
-		# The Image (fp) already exists, this simply makes a symlink in the
-		# cache to from the canonical syntax to the actual request path
+	def __setitem__(self, image_request, canonical_fp):
```

Comment:
It would be helpful if you could add some documentation or comments to the ImageCache class to make it a bit clearer what it is used for and how it works.  That would make it easier if contributors need to do anything with that part of the code.

If I come up with alternatives, I'll certainly let you know.  And glad the review was helpful!